pkg/kubernetes/persistentvolumeclaim/v1alpha1/persistentvolumeclaim.go

```diff
@@ -73,71 +73,78 @@ func (b *listBuilder) List() *pvcList {
 
 // Len returns the number of items present
 // in the PVCList
-func (p *pvcList) Len() int {
+func (p *PVCList) Len() int {
 	return len(p.items)
 }
 
 // ToAPIList converts PVCList to API PVCList
-func (p *pvcList) ToAPIList() *v1.PersistentVolumeClaimList {
-	plist := &v1.PersistentVolumeClaimList{}
+func (p *PVCList) ToAPIList() *corev1.PersistentVolumeClaimList {
```

Comment:
In addition to this, APIList() will be a method of *ListBuilder.
APIList will invoke `listBuilder.List()` internally

Reply:
Done